### PR TITLE
WFLY-11693, WFLY-11694 JGroups & Infinispan subsystem transformer tests for EAP 7.2

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.jboss.org/browse/WFLY-11693
https://issues.jboss.org/browse/WFLY-11694

### CODE REVIEW

File: clustering/infinispan/extension/src/test/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanTransformersTestCase.java [hunk 1]
```diff
@@ -97,6 +97,8 @@ private static InfinispanModel getModelVersion(ModelTestControllerVersion contro
                 return InfinispanModel.VERSION_4_0_0;
             case EAP_7_1_0:
                 return InfinispanModel.VERSION_5_0_0;
+            case EAP_7_2_0_TEMP:
+                return InfinispanModel.VERSION_8_0_0;
```
Comment (kabir) [inline clustering/infinispan/extension/src/test/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanTransformersTestCase.java]: @pferraro Now that the core upgrade has been merged this should be changed to just EAP_7_2_0

File: clustering/infinispan/extension/src/test/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanTransformersTestCase.java [hunk 2]
```diff
@@ -112,36 +114,23 @@ private static InfinispanModel getModelVersion(ModelTestControllerVersion contro
...
-                        formatArtifact("org.jboss.eap:wildfly-clustering-service:%s", version),
-                        formatArtifact("org.jboss.eap:wildfly-clustering-jgroups-spi:%s", version),
-                        // Following are needed for InfinispanSubsystemInitialization
-                        formatArtifact("org.jboss.eap:wildfly-clustering-jgroups-extension:%s", version),
-                        formatArtifact("org.jboss.eap:wildfly-connector:%s", version),
-                        formatArtifact("org.jboss.eap:wildfly-clustering-infinispan-spi:%s", version),
-                        formatArtifact("org.jboss.eap:wildfly-clustering-spi:%s", version),
-                };
             case EAP_7_1_0:
+            case EAP_7_2_0_TEMP:
```
Comment (kabir) [inline clustering/infinispan/extension/src/test/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanTransformersTestCase.java]: EAP_7_2_0

File: clustering/jgroups/extension/src/test/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsTransformersTestCase.java [hunk 3]
```diff
@@ -79,6 +79,8 @@ private static JGroupsModel getModelVersion(ModelTestControllerVersion controlle
                 return JGroupsModel.VERSION_4_0_0;
             case EAP_7_1_0:
                 return JGroupsModel.VERSION_5_0_0;
+            case EAP_7_2_0_TEMP:
+                return JGroupsModel.VERSION_6_0_0;
```
Comment (kabir) [inline clustering/jgroups/extension/src/test/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsTransformersTestCase.java]: EAP_7_2_0

File: clustering/jgroups/extension/src/test/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsTransformersTestCase.java [hunk 4]
```diff
@@ -92,18 +94,13 @@ private static JGroupsModel getModelVersion(ModelTestControllerVersion controlle
...
                 };
             case EAP_7_0_0:
-                return new String[] {
-                        formatEAP7SubsystemArtifact(version),
-                        formatArtifact("org.jboss.eap:wildfly-clustering-common:%s", version),
-                        formatArtifact("org.jboss.eap:wildfly-clustering-service:%s", version),
-                        formatArtifact("org.jboss.eap:wildfly-clustering-jgroups-spi:%s", version),
-                };
             case EAP_7_1_0:
+            case EAP_7_2_0_TEMP:
```
Comment (kabir) [inline clustering/jgroups/extension/src/test/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsTransformersTestCase.java]: EAP_7_2_0

File: clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanModel.java [hunk 5]
```diff
@@ -38,7 +38,7 @@
     VERSION_5_0_0(5, 0, 0), // WildFly 11, EAP 7.1
     VERSION_6_0_0(6, 0, 0), // WildFly 12
     VERSION_7_0_0(7, 0, 0), // WildFly 13
-    VERSION_8_0_0(8, 0, 0), // WildFly 14
+    VERSION_8_0_0(8, 0, 0), // WildFly 14, EAP 7.2
```
Comment (rhusar) [inline clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanModel.java]: ..also WildFly 15.

**Pull request merged**